geomesa-spark/geomesa-spark-jts/src/main/scala/org/apache/spark/sql/jts/JTSTypes.scala

```diff
@@ -44,26 +52,401 @@ object JTSTypes {
   )
 }
 
-private [spark] class PointUDT extends AbstractGeometryUDT[Point]("point")
-object PointUDT extends PointUDT
+class PointUDT extends AbstractGeometryUDT[Point]("point"){
+
+  // parquet definition:
+  // group.id(GeometryBytes.TwkbPoint)
+  //  .required(PrimitiveTypeName.DOUBLE).named(GeometryColumnX)
+  //  .required(PrimitiveTypeName.DOUBLE).named(GeometryColumnY)
+
+  override val sqlType: StructType = StructType(Seq(
```

Comment:
the python UDT is still declared as a struct of `'wkb': BinaryType`. So we get de/serialization like 

From the python side: 

> shapely.errors.WKBReadingError: Could not create geometry because of errors while reading input.